WordPress/src/jetpack/res/values/styles_toolbar.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <style name="Widget.LoginFlow.Toolbar" parent="Widget.MaterialComponents.Toolbar.Surface">
+        <item name="titleTextAppearance">@style/TextAppearance.App.Toolbar.Title</item>
```

Comment:
I'm not sure if I should override the login toolbar style here or add a `titleTextAppearance` (which doesn't exist right now) for the style in the login module. I'd appreciate any inputs on this 🙏.

Reply:
I think that for now what you did is okay and I would have done the same. Having said that I have a additional suggestion.

Suggestion (💡): Consider moving this style away from the `jetpack` to into the `main` `styles_toolbar.xml`. This way it will become more explicit that this `TextAppearance.App.Toolbar.Title` will be used for both the `WordPress` and `Jetpack` apps, just like it currently happens on the `WordPress.ToolBa` style. Wdyt?